README.md

```diff
@@ -101,7 +101,36 @@ class QuoteStartedTest extends TestCase
 
 You must first run the worker which will listen for events. 
 
-    `php artisan eventstore:worker`
+*None of the options are not required. By default it will run the persistance subscription with a timeout of 10 seconds and 1 parallel event at a time.*
```

Comment:
```suggestion
*None of the options are required. By default it will run the persistance subscription with a timeout of 10 seconds and 1 parallel event at a time.*
```